app/controllers/sufia/admin/admin_sets_controller.rb

```diff
@@ -1,7 +1,42 @@
 module Sufia
   class Admin::AdminSetsController < ApplicationController
+    include CurationConcerns::CollectionsControllerBehavior
     load_and_authorize_resource
     layout 'admin'
+    self.presenter_class = Sufia::AdminSetPresenter
+
+    def show
+      add_breadcrumb t(:'sufia.controls.home'), root_path
+      add_breadcrumb t(:'sufia.toolbar.admin.menu'), sufia.admin_path
+      add_breadcrumb t(:'sufia.admin.sidebar.admin_sets'), sufia.admin_admin_sets_path
+      add_breadcrumb 'View Set', request.path
+      super
+    end
+
+    # Override the default prefixes so that we use the collection partals.
+    def _prefixes
```

Comment:
Is this a public API of Rails? I'm wondering about `prepend_view_path` instead? (Not certain on the specfics). It also seams that the `@_prefixes ||=` is a possible point of confusion. Perhaps the following so we don't have incidental state related issues:

```ruby

DEFAULT_PREFIXES_FOR_COLLECTION_PARTIALS = ["sufia/admin/admin_sets", "collections", 'catalog'].freeze
def _prefixes
  DEFAULT_PREFIXES_FOR_COLLECTION_PARTIALS
end
```